src/ape/managers/query.py

```diff
@@ -128,6 +128,15 @@ def engines(self) -> dict[str, QueryAPI]:
 
     def _suggest_engines(self, engine_selection):
         return difflib.get_close_matches(engine_selection, list(self.engines), cutoff=0.6)
+    
+    def _get_skip_caching_plugins(self):
+        """
+        Get the list of query plugins to skip caching for from the configuration.
+        """
+        try:
+            return self.config_manager.query.skip_caching
```

Comment:
Adding this to the config would make it clearer how this feature works. Might also be nice to have the engine class override selectable via config, so it will always use that engine if set